<launch>

	<node pkg="voice_system" name="xf_tts_node" type="xf_tts_node" ns="catkin_ws" >
	<param name="speaker" type="string" value="xiaowanzi" />  <!--xiaofeng xiaowanzi xiaoxin-->
	<param name="pitch" type="string"  value="50" />         <!--音调控制 1～100-->
	<param name="appid" type="string"  value="5aaa223d" />   <!--科大讯飞帐号-->
	</node>

	<node pkg="voice_system" name="tuling_nlu_node" type="tuling_nlu_node" ns="catkin_ws" />


	<node pkg="voice_system" name="xf_asr_node" type="xf_asr_node" ns="catkin_ws" />

</launch>
